### ISSUE TITLE
Inability to use operators other than `eq` in the realtime filter within the `.from` argument

### ISSUE THREAD

**Issue by osaxma** [supabase-community/supabase-dart#90]
When I use any operator other than `eq` in the `.from` argument, I get the following error:
  ```
  relation "<schema>.<table>:<column>=<operator>.<value>" does not exist`
  ```
For the following operators:  `neq`, `lt`, `lte`, `gt`, and  `gte`.

For  instance:
- This works:
  ```dart
  supabaseClient.from("messages:is_seen=eq.false") 
  ```

- This does not work:
  ```dart
  supabaseClient.from("messages:is_seen=neq.false") 
  ```
  It gives the following error:
   ```
   relation "public.messages:is_seen=neq.false" does not exist
   ```

**Versions**
- supabase: ^0.2.13
- flutter: 2.8.1

**Comment by osaxma** [comment]
There's also another related issue when using `timestamptz` as a filter (I believe because it contains multiple `:`s and the colons are used as a separator). 

For instance:

- running this:

    ```dart
    supabaseClient.from("notifications:created_at=eq.$time")
    ```

    Gives the following error:
  
    ```
    column notifications.00 does not exist
    ```

**Comment by bdlukaa** [comment]
should it be `supabaseClient.from("messages:is_seen.neq.false") `?

**Comment by osaxma** [comment]
@bdlukaa I don't think so based on this example:

https://github.com/supabase-community/supabase-dart/blob/0ce7f7a8f1b39932e4097748bbe615ecbbc4ac10/lib/src/supabase_query_builder.dart#L53

Also, I just tried it and I got:
```
# for `eq`
relation "public.messages:is_seen.eq.false" does not exist
# for `neq`
relation "public.messages:is_seen.neq.false" does not exist
```

**Comment by osaxma** [comment]
It looks like the root of the problem is here:

https://github.com/supabase-community/supabase-dart/blob/0ce7f7a8f1b39932e4097748bbe615ecbbc4ac10/lib/src/supabase.dart#L56-L64


Obviously, only `eq` is taken into account here -- are there any reasons for that?

**Comment by phamhieu** [comment]
Hi @osaxma, right now the realtime server only support `eq` operator. 

https://github.com/supabase/realtime/blob/6df5ddc15cc79ac3eaa5e9af0bbbceea6129d2c2/README.md#supported-event-filters

**Comment by osaxma** [comment]
Hi @phamhieu, thanks for the info. 

Are there any plans to support the other operations soon?

**Comment by phamhieu** [comment]
> Are there any plans to support the other operations soon?

I don't think so. But feel free to create an issue on `realtime` repo if you want to get more information.

**Comment by osaxma** [comment]
@phamhieu thanks. I filed an issue that's referenced above.
